backgroundchanger/__main__.py

```diff
@@ -43,23 +58,30 @@ def do_wal(photo, do_colors=True):
     img = pywal.image.get(photo)
     if do_colors:
         colors = pywal.colors.get(img)
-        pywal.sequences.send(colors, sequences=True)
```

Comment:
no keyword argument as 'sequences' was being accepted; hence removed it

Reply:
Did the code change there?

Reply:
Yes, I removed the sequences keyword argument.

Reply:
With the _sequences=True_ , I was getting this error message:

`Traceback (most recent call last):
  File "/home/bishwajit/PycharmProjects/python-backgroundchanger/venv/bin/backgroundchanger", line 33, in <module>
    sys.exit(load_entry_point('backgroundchanger', 'console_scripts', 'backgroundchanger')())
  File "/home/bishwajit/PycharmProjects/python-backgroundchanger/backgroundchanger/__main__.py", line 96, in main
    do_wal(photo, set_colors)
  File "/home/bishwajit/PycharmProjects/python-backgroundchanger/backgroundchanger/__main__.py", line 61, in do_wal
    pywal.sequences.send(colors,sequences=True)
TypeError: send() got an unexpected keyword argument 'sequences'
`

Reply:
Oh yeah i meant the `pywal` code, but looks like it did in fact change. thanks for the fix